openml/datasets/functions.py

```diff
@@ -4,7 +4,7 @@
 import logging
 import os
 from pyexpat import ExpatError
-from typing import List, Dict, Union, Optional, cast
+from typing import List, Dict, Optional, Tuple, Union, cast  # noqa: F401
```

Comment:
Why do we need a noqa here?

Reply:
Indeed, that's not necessary. Will remove.